clippy_lints/src/assign_ops.rs

```diff
@@ -89,8 +90,14 @@ impl<'a, 'tcx> LateLintPass<'a, 'tcx> for AssignOps {
                                                 long
                                             ),
                                             format!("{} {}= {}", snip_a, op.node.as_str(), snip_r),
+                                            Applicability::Unspecified,
                                         );
-                                        db.span_suggestion(expr.span, "or", long);
+                                        db.span_suggestion_with_applicability(
+                                            expr.span, 
+                                            "or", 
+                                            long,
+                                            Applicability::Unspecified,
+                                            );
```

Comment:
weird indentation (for example)

Reply:
Is it `cargo fmt`-able? If not, I'm not sure what is the correct indentation.

I was doing it manually and used judgement call it is too far right or not and whether it is distinguishable as arguments or not...

Reply:
`rustfmt` will convert this to
```rust
db.span_suggestion_with_applicability(
    expr.span, 
    "or", 
    long,
    Applicability::Unspecified,
);
```